src/datasources/results/components/query-builders/query-steps/StepsQueryBuilder.test.tsx

```diff
@@ -9,14 +9,17 @@ describe('StepsQueryBuilder', () => {
 
     const containerClass = 'smart-element smart-query-builder';
     const workspace = { id: '1', name: 'Selected workspace' } as Workspace;
-    const stepsPath = ['path1', 'path2'];
+    const stepsPath = [
+      {label: 'Parent Path\Child Path1\Child Path2', value: 'Parent Path\nChild Path1\nChild Path2'},
+      {label: 'Another Path', value: 'Another Path'},
+    ]
     const status = ['PASSED', 'FAILED'];
 
     function renderElement(
       filter: string,
       workspaces: Workspace[] | null,
       stepStatus: string[],
-      stepsPath: string[],
+      stepsPath: Array<{ label: string; value: string }>,
```

Comment:
```suggestion
      stepsPath: StepPaths[],
```

Use the defined type instead